lib/compress/huf_compress.c

```diff
@@ -444,6 +444,7 @@ static int HUF_validateCTable(const HUF_CElt* CTable, const unsigned* count, uns
   int s;
   for (s = 0; s <= (int)maxSymbolValue; ++s) {
     bad |= (count[s] != 0) & (CTable[s].nbBits == 0);
+    bad |= (count[s] == 0) & (CTable[s].nbBits != 0);
```

Comment:
This makes sense right? We don't want the nbBits to be non-zero when the weight is zero. This gets rid of the other dictionary_loader fuzz failure but I'm not sure we want to actually do this. @terrelln, @Cyan4973?

Reply:
This would make nearly every CTable invalid. We construct dictionary CTables to have non-zero probability for every symbol, so that they always work.

With this change, any source that has any zero probability symbol will call the dictionary's CTable invalid.